README.md

```diff
@@ -1,26 +1,32 @@
-# [flutter_rust_bridge](https://github.com/fzyzcjy/flutter_rust_bridge): High-level memory-safe binding generator for Flutter/Dart <-> Rust
+<h1 align="center" style="border-bottom: none">
+    <b><a href="https://github.com/fzyzcjy/flutter_rust_bridge">flutter_rust_bridge</a></b>
+</h1>
 
-[![Rust Package](https://img.shields.io/crates/v/flutter_rust_bridge.svg)](https://crates.io/crates/flutter_rust_bridge)
-[![Flutter Package](https://img.shields.io/pub/v/flutter_rust_bridge.svg)](https://pub.dev/packages/flutter_rust_bridge)
-[![Stars](https://img.shields.io/github/stars/fzyzcjy/flutter_rust_bridge)](https://github.com/fzyzcjy/flutter_rust_bridge)
-[![CI](https://github.com/fzyzcjy/flutter_rust_bridge/actions/workflows/ci.yaml/badge.svg)](https://github.com/fzyzcjy/flutter_rust_bridge/actions/workflows/ci.yaml)
-[![Example](https://github.com/fzyzcjy/flutter_rust_bridge/actions/workflows/post_release.yaml/badge.svg)](https://github.com/fzyzcjy/flutter_rust_bridge/actions/workflows/post_release.yaml)
-[![Codacy Badge](https://api.codacy.com/project/badge/Grade/6afbdad19e7245adbf9e9771777be3d7)](https://app.codacy.com/gh/fzyzcjy/flutter_rust_bridge?utm_source=github.com&utm_medium=referral&utm_content=fzyzcjy/flutter_rust_bridge&utm_campaign=Badge_Grade_Settings)
+<h3 align="center">High-level memory-safe binding generator for Flutter/Dart <-> Rust</h3>
```

Comment:
Hmm I wonder does this work for `pub.dev` and `crates.io`? because the README will be sync into those two as well

Reply:
The `h1` and `h3` tag should work properly in Markdown files for both pub.dev and crates.io. When we publish the package, the README file will be rendered as HTML, which will just leave all HTML tags in place. Even complex tags like `table` were working well.

<img width="678" alt="image" src="https://user-images.githubusercontent.com/66480156/220846399-8a77b1f0-1b86-4bcb-8c5c-92dec6f45ca2.png">
<img width="553" alt="image" src="https://user-images.githubusercontent.com/66480156/220846681-94607dda-d9c4-40b0-a1fc-211aa11b04c8.png">